.gitignore

```diff
@@ -23,6 +23,9 @@
 # Ignore .dump
 /mylibnyc.dump
 
+# Ignore imported CSVs
+/data
```

Comment:
I'm concerned that I see a .csv in /data that got pushed and it part of this PR.  We might need to sanitize the repository again.  The good news, is that we won't have branch histories accumulating over time on top of this one.